Decred Wallet/Features/Navigation Menu/CustomTabMenuView.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  CustomTabMenuView.swift
+//  Decred Wallet
+//
+// Copyright (c) 2019 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+import UIKit
+
+class CustomTabMenuView: UIStackView {
+    var itemTapped: ((_ index: Int) -> Void)?
+    var activeTabIndex: Int = 0
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+    }
+    
+    required init(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+    
+    convenience init(menuItems: [MenuItem], frame: CGRect) {
+        self.init(frame: frame)
+        self.axis = .horizontal
+        self.layer.backgroundColor = UIColor.white.cgColor
```

Comment:
Without setting the distribution of this stackview to fillEqually, the following is seen on an iPhone 11:
<img width="301" alt="Screenshot 2019-11-09 at 7 46 17 AM" src="https://user-images.githubusercontent.com/18400051/68524489-c8460180-02c7-11ea-816d-4156fe9e6d0e.png">

With the distribution set to fillEqually:
<img width="304" alt="Screenshot 2019-11-09 at 7 47 03 AM" src="https://user-images.githubusercontent.com/18400051/68524491-cb40f200-02c7-11ea-9d17-f85aff1bff08.png">

Reply:
resolved in 2766f8f